inst/ms/aggreCAT.qmd

```diff
@@ -1400,16 +1521,13 @@ green_turtles <-
 
 We can then apply multiple aggregation methods, using the same approach
 implemented for aggregation of the [focal_claims]{.data} dataset
-(@lst-multi-method-workflow-non-supp), with aggregated Confidence Scores
+(@lst-BYO-data-aggregate), with aggregated Confidence Scores
 shown in @tbl-BYO-data-aggregate. Note that because the judgements are
-absolute values rather than probabilities, we set the [percent\\_toggle]{.arg} argument for each
-aggregation wrapper function to [FALSE]{.val}:
+absolute values rather than probabilities, we set the
+[percent\_toggle]{.arg} argument for each aggregation wrapper function
+to [FALSE]{.val}: (DAVID/AAron to check.. )
 
-```{r}
-#| label: lst-BYO-data-aggregate
-#| prompt: true
-#| results: false
-#| message: false
+```{#lst-BYO-data-aggregate .r lst-cap='Bring your own data: non-probablistic values'}
```

Comment:
This code chunk showing the agg method calls isn't showing, just the results

Reply:
The code is shown as a 'listing', 'listing 2' on page 36 of the PDF file, as I was trying to free up space in the main text, and wasn't sure if it was overkill having all of the worked example case study stuff in there. Let's discuss tomorrow about whether to make it appear in the main text rather than a listing @Doi90 @psychtek . Same goes for Listing 1.

Reply:
Oh I think I see:

> Score for over 3000 SBS research claims for phase one and **X** SBS
claims for phase two of the DARPA SCORE project.

- [ ] Total claims for whole project please somebody?

Reply:
For the paper, ok to leave as is, but for using the PDF/paper as vignette will want to move the code chunk into the main-text rather than having it as a listing.